sox.dialog.js

```diff
@@ -141,21 +138,12 @@
 
           const $feature = $soxSettingsDialogFeatures.find('input#' + name).parent();
           $feature.append($settingsToggle);
-
-          if ($div.has('.sox-sprite-info').length) {
-            $info.after($settingsDiv);
-          } else {
-            $feature.append($settingsDiv);
-          }
+          $div.has('.sox-sprite-info').length ? $info.after($settingsDiv) : $feature.append($settingsDiv);
```

Comment:
You could convert this to a 2-line if instead of using a ternary like I mentioned in the other comment maybe?